src/main/java/seedu/address/logic/commands/profile/AddProfileCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands.profile;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_OPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.profile.Profile;
+
+/**
+ * Adds a profile to the address book.
+ */
+public class AddProfileCommand extends ProfileCommand {
+
+    public static final String COMMAND_OPTION = "a";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " " + PREFIX_OPTION + COMMAND_OPTION
+            + ": Adds a profile to the address book. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "PHONE "
+            + PREFIX_EMAIL + "EMAIL\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_OPTION + COMMAND_OPTION + " "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_PHONE + "98765432 "
+            + PREFIX_EMAIL + "johnd@example.com "
+            + PREFIX_TAG + "friends "
+            + PREFIX_TAG + "owesMoney";
```

Comment:
should this be removed since tag is not implemented?

Reply:
Tags are implemented in this milestone.